### PR TITLE
feat(workflow): Add singular ignore option

### PR DESCRIPTION
**Pull request by dcramer**
This adds a quick action to "ignore until its been seen one more time."

I've found this case to be fairly valuable, and is a better option than marking it as resolved when all you really care about is "this occurance didnt matter, but let me know if it continues to happen."

### CODE REVIEW

**Comment by dcramer** [comment]
screen shot 2018-10-22 at 12 53 52 pm

File: src/sentry/static/sentry/app/components/actions/ignore.jsx [hunk 1]
```diff
@@ -173,7 +173,7 @@ export default class IgnoreActions extends React.Component {
                   return (
                     <li className="dropdown-submenu" key={count}>
                       <DropdownLink
-                        title={t('%s times', count.toLocaleString())}
+                        title={tn('one time', '%s times', count)}
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/actions/ignore.jsx]: Should this be consistent with the other menu items?

```suggestion
                        title={tn('1 time', '%s times', count)}
```

Comment (dcramer) [reply src/sentry/static/sentry/app/components/actions/ignore.jsx]: My personal preference is spelling out one (I think theres some English best practice there), but I'm not opposed.

Anyone else have thoughts?

**Review by billyvg — APPROVED** [review]
I don't have a strong preference here

**Comment by dcramer** [comment]
Cool will merge if GH starts working and we can always change in the future.

File: src/sentry/static/sentry/app/components/actions/ignore.jsx (hunk 1 above)
Comment (benvinegar) [reply src/sentry/static/sentry/app/components/actions/ignore.jsx]: I prefer `1 time` for consistency

**Pull request merged**